content/_help/verify-your-identity/accepted-state-issued-identification._en.md

```diff
@@ -1,33 +1,80 @@
 ---
 layout: help
-title: Accepted state-issued identification
+title: Accepted identification documents
 category: verify-your-identity
 permalink: /help/verify-your-identity/accepted-state-issued-identification/
 order: 2
 redirect_from:
   - /en/help/verify-your-identity/accepted-state-issued-identification/
 ---
 
-At this time, only the following valid state-issued identification is accepted: 
+<ul class="usa-icon-list usa-icon-list--size-md">
+  <li class="usa-icon-list__item">
+    <div class="usa-icon-list__icon text-green">
+      <svg class="usa-icon" aria-hidden="true" role="img">
+        <use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#check_circle"></use>
+      </svg>
+    </div>
+    <div class="usa-icon-list__content">
+      <strong>Driver’s license</strong> from all 50 states, the District of Columbia (DC), and other US territories (Guam, US Virgin Islands, American Samoa, Mariana Islands and Puerto Rico).
+    </div>
+  </li>
+    <li class="usa-icon-list__item">
```

Comment:
indentation is off here
```suggestion
  </li>
  <li class="usa-icon-list__item">
```

Should we make partials for this? Off the top of my head, I would make ones for:
- SVG icon
- The icon list

We could pass in an array of strings from the page?

ex

```
list:
  - **Driver’s license** from all 50 states, the District of Columbia (DC), and other US territories (Guam, US Virgin Islands, American Samoa, Mariana Islands and Puerto Rico).
  -  **Non-driver’s license state-issued ID card.** This is an identity document issued by the state, the District of Columbia (DC), or US territory that asserts identity but does not give driving privileges.
```

and then

```
{% include icon_list.html list=page.list %}
```

Reply:
Was talking with @charleyf , he proposed same thing.

Reply:
@zachmargolis , refactored, looks like the icon-list is affected by style .page-content__prose ul resulted in a padding-left : 3inch.(@charleyf  pointed this out).

Fixing it with this with inline style for now.